src/image/image.js

```diff
@@ -175,14 +175,24 @@ p5.prototype.saveCanvas = function() {
 
   // copy arguments to array
   const args = [].slice.call(arguments);
-  let htmlCanvas, filename, extension;
+  let htmlCanvas, filename, extension, temporaryGraphics;
 
   if (arguments[0] instanceof HTMLCanvasElement) {
     htmlCanvas = arguments[0];
     args.shift();
   } else if (arguments[0] instanceof p5.Element) {
     htmlCanvas = arguments[0].elt;
     args.shift();
+  } else if (arguments[0] instanceof p5.Framebuffer) {
+    const framebuffer = arguments[0];
+    temporaryGraphics = createGraphics(framebuffer.width, framebuffer.height);
```

Comment:
Oh we might also need to set the graphic's pixel density to match after this line too. Otherwise this implementation is looking good!